README.md

```diff
@@ -83,14 +83,13 @@ Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
 
 (open a pull request if you want your project to be added here)
 
-- https://github.com/paritytech/polkadot
-- https://github.com/paritytech/substrate
-- https://github.com/sigp/lighthouse
-- https://github.com/golemfactory/golem-libp2p
-- https://github.com/comit-network
-- https://github.com/rs-ipfs/rust-ipfs
-- https://github.com/marcopoloprotocol/marcopolo
-- https://github.com/ChainSafe/forest
-- https://github.com/ipfs-rust/ipfs-embed
-- https://www.actyx.com/developers/
-- https://github.com/starcoinorg/starcoin
+- [COMIT](https://github.com/comit-network/xmr-btc-swap) - Bitcoin–Monero Cross-chain Atomic Swap.
+- [Forest](https://github.com/ChainSafe/forest) - An implementation of Filecoin written in Rust.
+- [Lighthouse](https://github.com/sigp/lighthouse) - Ethereum consensus client in Rust.
+- [Locutus](https://github.com/freenet/locutus) - Global, observable, decentralized key-value store.
+- [Starcoin](https://github.com/starcoinorg/starcoin) - A smart contract blockchain network that scales by layering.
+- [Subspace](https://github.com/subspace/subspace) - Subspace Network reference implementation
+- [Substrate](https://github.com/paritytech/substrate) - Framework for blockchain innovation.
+- [ipfs-embed](https://github.com/ipfs-rust/ipfs-embed) - A small embeddable ipfs implementation.
```

Comment:
This is currently only driven by Actyx, it is just not in our github organisation (and the main Actyx codebase is private). Make of that what you want :-)

Reply:
Hi Roland, thanks for the input. You mean `ipfs-embed` right? Would you like to add that it's maintained by Actyx with a link to your website?

Reply:
If you think it adds value to this list, then a simple “(used e.g. by Actyx)” on the ipfs-embed line would suffice as far as I’m concerned.

Reply:
ok updated :)